public/main.js

```
@@ -4,135 +4,134 @@
 }
 
 var numberCells = document.getElementsByClassName("cellNumber");
 
-for (var i = 0 ; i < numberCells.length; i++) {
-	numberCells[i].addEventListener("click", selectRowNumber);
+for (var i = 0; i < numberCells.length; i++) {
+  numberCells[i].addEventListener("click", selectRowNumber);
 }
 
 var priceCells = document.getElementsByClassName("cellPrice");
 for (var i = 0; i < priceCells.length; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)